react/utils/client/index.ts

```diff
@@ -24,29 +25,28 @@ export const getState = (runtime: RenderRuntime) => {
 }
 
 export const getClient = (runtime: RenderRuntime) => {
-  const {graphQlUri, account, workspace} = runtime
+  const {account, workspace} = runtime
```

Comment:
We will need the `graphQlUri` in other parts of the runtime, like fetching messages. @brunoabreu @tlgimenes how do we go around that?

Reply:
I have no idea. I think you and @brunoabreu  can make a patch to this branch solving this problem